README.md

```diff
@@ -15,10 +15,12 @@ For information about the Design please visit [design](docs/design.md) page.
 
 ## Setup
 
-Install latest rustc.
-> Note: The version of rustc that can be installed via apt is outdated.
+Install latest rustc via `rustup` if building on Ubuntu Jammy.
+Skip this step if building on Ubuntu Noble as `cargo` and `rustc` will be
```

Comment:
This is cool! When did that happen?

Reply:
The debs are available in Jammy too but the versions are very dated so we install them via `rustup` to compile zenoh. Noble ships with cargo/rustc `1.75.0` which works out of the box for our use case.